triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TokenManagement.scala

```diff
@@ -64,9 +64,9 @@ object TokenManagement {
   // token. By construction we ensure that there will always be two
   // components and that the first component is a syntactically valid
   // party identifier (see 'findCredentials').
-  def decodeCredentials(key: SecretKey, creds: UserCredentials): (Party, Password) = {
+  def decodeCredentials(key: SecretKey, creds: UserCredentials): (String, String) = {
```

Comment:
Why did you make this change?

Reply:
Just making things easier for now while I try to get things to work. This encryption interface will need to change a bit anyway as it's kind of awkward to use.

Reply:
Fair enough, not quite sure why this is easier and regardless of how we change the interface I expect we’ll pass around parties and passwords but I’m fine changing things temporarily.